.github/workflows/enos-run.yml

```diff
@@ -25,10 +25,11 @@ jobs:
       fail-fast: false # don't fail as that can skip required cleanup steps for jobs
       matrix:
         include:
-          - filter: 'integration test:cli_ui'
-          - filter: 'e2e_aws'
-          - filter: 'e2e_static'
-          - filter: 'e2e_static_with_vault'
+          - filter: 'integration test:cli_ui builder:crt'
```

Comment:
Needed to move this `builder:crt` filter higher up since this new test doesn't need a boundary cluster spin up by enos.